lib/views/help/unhappy.html.erb

```diff
@@ -40,7 +40,7 @@
 
 <p>
   <% if !@info_request.nil? %>
-    <%= link_to "Request an internal review", show_response_no_followup_path(:id => @info_request.id, :incoming_message_id => nil) + "?internal_review=1#followup", :class => 'link_button_green request-internal-review-action' %> and then write a message asking the authority to review your request.
+    <%= link_to "Request an internal review", new_request_incoming_followup_path(:id => @info_request.id, :incoming_message_id => nil) + "?internal_review=1#followup", :class => 'link_button_green request-internal-review-action' %> and then write a message asking the authority to review your request.
```

Comment:
As `incoming_message_id` is `nil`, I think the correct route should be:
```suggestion
    <%= link_to "Request an internal review", new_request_followup_path(:request_id => @info_request.id, :internal_review => '1', :anchor => 'followup'), :class => 'link_button_green request-internal-review-action' %> and then write a message asking the authority to review your request.
```

Reply:
Oh, durr! Just saw the key and didn't even notice it was set to `nil` 🤦‍♂